index.html

```diff
@@ -37,6 +39,147 @@ <h2 class="connect">LET’S CONNECT</h2>
 
 </div>
 
+<section class="works">
+  <article class="work1">
+    <img src="work1-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Tonic</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="program-list">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+  <article class="work2">
+    <img src="work2-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Multi-Post Stories</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="program-list">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+  <article class="work3">
+    <img src="work3-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Tonic</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="program-list">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+  <article class="work4">
+    <img src="work4-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Multi-Post Stories</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="program-list">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+</section>
+
+<section class="about-section">
```

Comment:
- [ ] Kindly style well work section and about section
![Capturefix](https://user-images.githubusercontent.com/81444338/176997112-dbdf3494-f60e-4ad2-bd0b-730bf6bb1700.PNG)